### PR TITLE
Use publish plugin to build MPK

### PR DESCRIPTION
**Pull request by dahfjkg**

### CODE REVIEW

File: marketplace/description.html [hunk 1]
```diff
@@ -0,0 +1 @@
+This module ensures tracking of the changes made in the system. We support 2 major versions of this module: - Version 9 is compatible with Atlas UI 3. It is only available in Mendix 9. It should be used for new apps created with Mendix 9. - Version 8 is compatible with Atlas UI 2. It is available in both Mendix 8 and 9.
```
Comment (hugomendix) [inline marketplace/description.html:1]: Maybe you could use HTML tags to layout the bullet items more nicely.

**Review by hugomendix — CHANGES_REQUESTED** [review]

**Comment by dahfjkg** [comment]
rebased on `dat/timezone-logging` ⬆️

File: build.gradle [hunk 2]
```diff
@@ -1,36 +1,88 @@
...
+
+task prepareTest {
+    dependsOn 'clean'
+    dependsOn 'copyAllToUserlib'
+
+    tasks.findByName('copyAllToUserlib').mustRunAfter 'clean'
+}
+
+exportModule.dependsOn prepareMpk
+mxBuild.dependsOn prepareTest
```
[the remark this answers is not in the document]
Comment (dahfjkg) [reply build.gradle:88]: `exportModule` comes from the plugin, so `exportModule.dependsOn prepareMpk` should be enough. I will add the rest.

**Comment by hugomendix** [comment]
LGTM

**Review by hugomendix — DISMISSED** [review]

**Review by hugomendix — APPROVED** [review]

**Review by hugomendix — APPROVED** [review]

**Pull request merged**